src/Symfony/Component/Yaml/Inline.php

```diff
@@ -231,6 +231,9 @@ public static function parseScalar($scalar, $delimiters = null, $stringDelimiter
 
             if (null !== $delimiters) {
                 $tmp = ltrim(substr($scalar, $i), ' ');
+                if ('' === $tmp) {
+                    throw new ParseException(sprintf('Unexpected end of line, expected one of "%s".', implode($delimiters)));
```

Comment:
when merging on 3.4:
`throw new ParseException(sprintf('Unexpected end of line, expected one of "%s".', implode($delimiters)), self::$parsedLineNumber + 1, $scalar, self::$parsedFilename);`